src/ui/fuzzy_finder.rs

```diff
@@ -0,0 +1,31 @@
+use fuzzy_finder::item::Item;
+use std::io::Write;
+
+pub fn create(issues: Vec<String>) -> String {
+    // Create a list of items from the list of issues
+    let items: Vec<Item<String>> = issues
+        .iter()
+        .map(|issue| Item::new(issue.to_string(), issue.to_string()))
+        .collect();
+
+    // Prompt the user to select an issue from the list of issues
+    let items_count = items.len();
+
+    let find_result = match fuzzy_finder::FuzzyFinder::find(items, items_count.try_into().unwrap())
+    {
+        Ok(result) => result,
+        Err(e) => {
+            std::io::stdout().flush().unwrap();
+            panic!("Failed to find result: {}", e);
+        }
+    };
+    let item = match find_result {
+        Some(result) => result,
+        None => {
+            panic!("Invalid result");
+        }
+    };
+    println!();
+
+    return item;
```

Comment:
```suggestion
    match fuzzy_finder::FuzzyFinder::find(items, items_count.try_into().unwrap()) {
        Ok(Some(result)) => result,
        Ok(None) => {
            panic!("Invalid result");
        }
        Err(e) => {
            std::io::stdout().flush().unwrap();
            panic!("Failed to find result: {}", e);
        }
    }
```

Simplification (and removal of `println!` that is included now in `main.rs`.

Reply:
I think maybe the `println!` make more sense inside the `ui`? But for sure should be in only 1 place.

Reply:
Mhhh, it could go on either. My reasoning is that this function should just do the selection itself, and the printf is a further thing done, but it can be thought of part of the finder experience though.